spec/swagger.yaml

```diff
@@ -236,6 +240,73 @@ paths:
             -d '{"refreshToken": "eyJhbGciO..."}'
 
 #
+# Accounts & Projects
+#
+
+  '/accounts-api/v2/accounts/{accountUid}/projects':
+    get:
+      summary: List Projects
+      description: |
+        Return the list of projects for the account
+      tags:
+        - Account & Projects
+      parameters:
+        - name: accountUid
+          in: path
+          required: true
+          type: string
+        - name: projectNameFilter
+          in: query
+          required: false
+          type: string
+          description: Substring search of the name of the project. Search is case insensitive.
+        - name: includeArchived
+          in: query
+          required: false
+          type: boolean
+          description: Indicator whether or not archived projects should be returned. Defaults to false if not specified.
+        - name: offset
+          in: query
+          required: false
+          type: integer
+          description: Standard Smartling limit and offset to paginate through results.
+        - name: limit
+          in: query
+          required: false
+          type: integer
+          description: Standard Smartling limit and offset to paginate through results.
```

Comment:
Just curious, does the endpoint applies default `limit` value when nothing is specified and/or claps its maximum value? Do we want to surface that info in the docs? Probably it's not critical for this endpoint but I would like to understand our general approach for documenting such things.

Reply:
added default limit